flytekit/core/data_persistence.py

```diff
@@ -128,7 +128,7 @@ def __init__(
         self._local = fsspec.filesystem(None)
 
         self._data_config = data_config if data_config else DataConfig.auto()
-        self._default_protocol = get_protocol(raw_output_prefix)
+        self._default_protocol = get_protocol(str(raw_output_prefix))
```

Comment:
curious, why do we need this?

Reply:
Some files are calling this method but pass a `Path` to raw_output_prefix